gss-cb-signalling.xml

```diff
@@ -233,9 +233,9 @@ Whenever both the initiator and acceptor have provided input_channel_bindings to
           <t>
 Whenever the acceptor application has a) provided channel bindings to GSS_Accept_sec_context(), and b) not indicated support for the ret_channel_bound_flag flag, then the mechanism MUST fail to establish a security context if the initiator did not provide channel bindings data. This requirement is critical for security purposes, to make applications predating this document secure, and this requirement reflects actual implementations as deployed.</t>
           <t>
-Whenever the initiator application has a) provided channel bindings to GSS_Init_sec_context(), and b) not indicated support for the ret_channel_bound_flag flag, then the mechanism MUST NOT fail to establish a security context just because the acceptor failed to provide channel bindings data. This requirement is for interoperability purposes, and reflects actual implementations that have been deployed.</t>
+Whenever the initiator application has a) provided channel bindings to GSS_Init_sec_context(), and b) not indicated support for the ret_channel_bound_flag flag, then the mechanism MUST NOT fail to establish a security context just because the acceptor failed to provide channel bindings data unless the channel bindings are used for key derivation. This requirement is for interoperability purposes, and reflects actual implementations that have been deployed.</t>
```

Comment:
This change basically changes the MUST NOT to a SHOULD NOT.  It would be better to make that change than to call out key derivation as the only possible reason.  Perhaps there are other reasons (that I haven't thought of) that a mech can't satisfy the requirement.

Reply:
Incidentally, it's also okay with me to keep the MUST NOT and just say that new mechanisms have to be able to support these semantics (i.e. you cannot use channel bindings for key derivation, because such a mech would render it impossible to phase in channel bindings to an application protocol).

Reply:
Alright.  Given the choice, does anyone have feelings about MUST NOT vs. SHOULD NOT then?  Using the channel bindings in such a manner seems reasonable to me, but it also weakens the requirements and I don't know whether the trade-off is worth it.